README.md

```diff
@@ -81,6 +84,26 @@ to get document access.
 - [bit.ly/kueue-controller-design](https://bit.ly/kueue-controller-design)
 presents the detailed design of the controller.
 
+## Roadmap
+
+This is a high-level overview of the main priorities for 2023, in expected order of release:
+
+- Job preemption to reclaim borrowed quota and to accommodate high priority jobs [#83](https://github.com/kubernetes-sigs/kueue/issues/83), this is planned for v0.3
+- More advanced allocation strategies for better resource sharing [#312](https://github.com/kubernetes-sigs/kueue/issues/312)
+- Integration with cluster-autoscaler for guaranteed resource provisioning
+- Integration with common custom workloads [#74](https://github.com/kubernetes-sigs/kueue/issues/74):
```

Comment:
Why put this in the last place? I think this should be a higher priority.

Reply:
This is ordered according to our current involvement/capacity. We certainly want this, but there isn't anybody working on it at the moment.

Reply:
Okay, got it. Thank you for explaining this, Aldo.